SDK/Hardware/HA4IoT.Hardware.Knx/KnxController.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Diagnostics;
+
+namespace HA4IoT.Hardware.Knx
+{
+    public class KnxController
+    {
+        //var knxController = new KnxController("192.168.1.123","8150");
+        //var lamp = new HA4IoT.Actuators.Lamp(new ComponentId("Lamp1"), knxController.CreateDigitalJoinEndpoint("d1"));
+        SocketClient _socketClient;
+
+        public KnxController(string hostName, string portNumber)
+        {
+            try
+            {
+                _socketClient = new SocketClient();
+                Debug.WriteLine(">>>>>>>>>> knx <<<<<<<<<<");
```

Comment:
Please consider using the "Log" class from the project. It will write to the Debug output (if attached) and it will send the trace through the UDP sender. Then you will get all messages using the TraceViewer App.